ioctl/newcmd/action/action.go

```diff
@@ -171,6 +171,41 @@ func RegisterWriteCommand(client ioctl.Client, cmd *cobra.Command) {
 	registerPasswordFlag(client, cmd)
 }
 
+// GetWriteCommandFlag returns action flags for command
+func GetWriteCommandFlag(cmd *cobra.Command) (gasPrice, signer, password string, nonce, gasLimit uint64, assumeYes bool, err error) {
```

Comment:
IMO, creating a shared func `GetWriteCommandFlag` under `action.go` is strange. Doing argu reading job in corresponding cmd should be fine.

Reply:
Other commands( sub action or contract commands) will  get the write command flags, so combined them.

Reply:
I think both of the reasons make sense, how about creating a private function `getWriteCommandFlag` in both `action` and `contract` package, they can shared the function with sub commands and can't be acccessed by other packages.